diff --git a/es-core/src/resources/Font.cpp b/es-core/src/resources/Font.cpp
index aea473e..1708e96 100644
--- a/es-core/src/resources/Font.cpp
+++ b/es-core/src/resources/Font.cpp
@@ -384,7 +384,7 @@ std::vector<std::string> getFallbackFontPaths()
 
 	// TODO
 	const char* paths[] = {
-		"/usr/share/fonts/truetype/droid/DroidSansFallbackFull.ttf",// japanese, chinese, present on Debian
+		"/usr/share/fonts/truetype/droid/DroidSans.ttf",
 		":/fontawesome_webfont.ttf",
 		 "/usr/share/fonts/truetype/fontawesome-webfont.ttf",
 	};
